### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by ashacekar** [gray-stanton/spamGAN#2]
Hi, Gray-Stanton
I m not able to understand dataset used here and divisions used into multiple txt. as they are unavailable. Could i know what is the dataset used and how u divide it into review, label, vocab or other tags used?

**Comment by gray-stanton** [comment]
Hi ashcekar,
Prior to any real processing, the data/opspam_reviews and data/opspam_labels.txt are the cleaned version of the Ott dataset. The various data/chicago_* files are the unlabeled datasets we extracted from tripadvisor. The labeled dataset are deduplicated and split once into train/test. That training data as well as the unlabeled reviews are further subdivided as outlined in the paper. 

Hope that helps!

**Comment by ashacekar** [comment]
Thank You for the help. Could u upload the data folder to this repository as it's not there?

**Comment by ashacekar** [comment]
Thx buddy
